.github/workflows/ci.yml

```diff
@@ -998,6 +998,8 @@ jobs:
     name: PHPUnit (PHP ${{ matrix.php }}) (Symfony lowest)
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    env:
+      SYMFONY_PHPUNIT_VERSION: '9.5'
```

Comment:
This will solve the issue for CI, but not for locally executed runs?

Reply:
at least it runs the tests, no idea how to fix this otherwise, I wish we could just use phpunit 10 without relying on the symfony bridge.